listenerutil/listener.go

```diff
@@ -146,6 +146,7 @@ PASSPHRASECORRECT:
 				badCiphers = append(badCiphers, cipherStr)
 			}
 		}
+		// TODO: change this to not use a vault-specific error
```

Comment:
Is this happening :P? Maybe in a separate PR?

Reply:
this is mostly for future reference, though it is such a small change that I *could* add it in this pr?

Reply:
Go for it IMO